Formula/llvm.rb

```diff
@@ -136,16 +147,21 @@ def install
     builtins_cmake_args = []
 
     # Skip the PGO build on HEAD installs or non-bottle source builds
-    pgo_build = build.stable? && build.bottle?
+    # pgo_build = build.stable? && build.bottle?
+    pgo_build = false # deactivate while testing
 
     if OS.mac?
       args << "-DLLVM_BUILD_LLVM_C_DYLIB=ON"
       args << "-DLLVM_ENABLE_LIBCXX=ON"
       args << "-DDEFAULT_SYSROOT=#{macos_sdk}" if macos_sdk
       runtimes_cmake_args << "-DCMAKE_INSTALL_RPATH=#{rpath}"
+      # Disable builds for OSes not supported by the CLT SDK.
+      runtimes_cmake_args += %w[I WATCH TV].map { |os| "-DCOMPILER_RT_ENABLE_#{os}OS=OFF" }
```

Comment:
> I WATCH TV

😄 

What's changed here though to need this?

Reply:
This https://github.com/llvm/llvm-project/commit/a680c212cb213bf73be7d3e2ee919fdc743cef0c seems to lead to https://github.com/Homebrew/homebrew-core/pull/106925#issuecomment-1221304469.

(Yes, I chuckled at `I WATCH TV` too. Though really only the `I` is potentially needed right now.)

Reply:
Actually, I'm not sure why this is failing. [`find_darwin_sdk_version`](https://github.com/llvm/llvm-project/blob/804d4594cbe217ae817b6786b0e9965283f78aa2/compiler-rt/cmake/Modules/CompilerRTDarwinUtils.cmake#L46) runs `xcrun --sdk iphonesimulator --show-sdk-version` and checks that it returns `0`. I tried this on an ARM runner and it seems to work.

Is it because we set `DEVELOPER_DIR` or something?

Reply:
I suppose we could just require Xcode to build this again...

Reply:
I'll check shortly.

> I suppose we could just require Xcode to build this again...

I'd rather not.

Reply:
You probably won't be able to submit apps for those platforms when built with Homebrew LLVM so I don't think this is an issue.

Reply:
> You probably won't be able to submit apps for those platforms when built with Homebrew LLVM so I don't think this is an issue. Was just mostly interested why it broke.

Maybe not, but it might be nice to have access to sanitisers, etc, for debugging.

> Homebrew is also a macOS package manager and offer no iOS support in anything else we ship.

Can we force this to use the Xcode SDK if it's available but fall back to the CLT and disable iOS builds if not?

Reply:
We could but it's tricky to selectively apply it to just iOS etc SDKs.

Setting `DARWIN_iphonesimulator_OVERRIDE_SDK_VERSION` (etc) is possible however. It seems like before it might have just auto-disabled if the SDK is not available (maybe? - need to check) so this feels like a bit of a regression anyway and I don't think it should be a fatal error. Maybe worth comparing the bottle with a CLT-only build at some point.